Examples/Tests/uniform_plasma/inputs_3d

```diff
@@ -0,0 +1,89 @@
+# algo
```

Comment:
As discussed offline: this could be moved to a folder `nci_psatd_stability` and consolidated with other similar tests (e.g. Galilean).